pkg/scheduler/util/utils.go

```diff
@@ -103,3 +105,26 @@ func MoreImportantPod(pod1, pod2 *v1.Pod) bool {
 	}
 	return GetPodStartTime(pod1).Before(GetPodStartTime(pod2))
 }
+
+// GetPodServices gets the services that have the selector that match the labels on the given pod.
+func GetPodServices(serviceLister v1listers.ServiceLister, pod *v1.Pod) ([]*v1.Service, error) {
```

Comment:
we are trying to get rid of the util package in the scheduler, please don't add any logic here, this should stay in the listers package.

Reply:
We're trying to remove manual changes to the listers interfaces. A helper method makes sense for this functionality, but the location is open for discussion (but shouldn't really be in generated packages)

Reply:
yeah, I meant moving this as is out of scheduler/util and place it in scheduler/listers

Reply:
> yeah, I meant moving this as is out of scheduler/util and place it in scheduler/listers

ok, I'll move it to scheduler/listers

Reply:
Done.
Personal opinion: moving to scheduler/listers directly seems not good yet, GetPodServices doesn't belong to PodLister or NodeInfoLister. Maybe it could be refactored as a function of ServiceLister in the future.

Reply:
Right, we will move it to the exact framework plugin that uses it once that plugin is ready.